internal/webhooks/runtime/extensionconfig_webhook.go

```diff
@@ -218,5 +218,13 @@ func validateExtensionConfigSpec(e *runtimev1.ExtensionConfig) field.ErrorList {
 			"must be defined",
 		))
 	}
+
+	if _, err := metav1.LabelSelectorAsSelector(e.Spec.NamespaceSelector); err != nil {
+		allErrs = append(allErrs, field.Invalid(
+			specPath.Child("NamespaceSelector"),
```

Comment:
```suggestion
			specPath.Child("namespaceSelector"),
```
also above, please check others